xla/service/gpu/cudnn_fused_conv_rewriter.cc

```diff
@@ -94,8 +94,12 @@ bool IsNonDepthwiseConvCustomCall(const HloInstruction* instr) {
 // Note that as of writing, xla_gpu_use_runtime_fusion is disabled by default
 // due to apparent bugs in cudnn 8.9.0.  See debug_options_flags.cc for details.
 bool ShouldUseCudnnRuntimeFusion(const DebugOptions& debug_opts,
-                                 se::CudaComputeCapability cc) {
-  return debug_opts.xla_gpu_use_runtime_fusion() && cc.IsAtLeast(7, 5);
+                                 se::GpuComputeCapability cc) {
+  auto pgcc = std::get_if<se::CudaComputeCapability>(&cc);
+  if(pgcc != nullptr)
+    return debug_opts.xla_gpu_use_runtime_fusion() && pgcc->IsAtLeast(7, 5);
+  else
+    return true;
```

Comment:
As I understand, runtime fusion simply means that kernels are being compiled on-demand at the time of call? ROCm convolutions, both fused and unfused, have been working this way since day one.

Reply:
That makes sense. I got confused by the name of the flag. It is cuDNN specific and should have that in the name, but that's outside of the scope of this PR of course.